### PR TITLE
[OM-99261] Update prometurbo README in GitHub

### PR DESCRIPTION
**Pull request by ading1977**
Please review the README here:

* [Top level README](https://github.com/ading1977/prometurbo/tree/readme)
* [Deployment README](https://github.com/ading1977/prometurbo/tree/readme/deploy)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,58 @@
...
+i.e., DIF Probe) will access the REST API, convert the JSON output to Turbonomic DTO to be consumed by Turbonomic
+server. This enables Turbonomic to collect and analyze Prometheus metrics and make intelligent decisions about
+application scaling, placement and optimization.
+
+<img width="800" alt="appmetric" src="https://user-images.githubusercontent.com/10012486/80402653-34bfb780-888c-11ea-82f8-f102452047ff.png">
+
+To configure the Prometheus server and map query results into applications and metrics, you need to create the following
+custom resources in the Kubernetes cluster:
+
+* [PrometheusQueryMapping](https://pkg.go.dev/github.com/turbonomic/turbo-metrics@v0.0.0-20230222215340-3cdff28ffdaf/api/v1alpha1#PrometheusQueryMapping)
```
Comment (amd-ibm) [inline README.md:14]: nit: seems like there should be a colon or hyphen separating the item that needs to be deployed from the description of that item.

File: deploy/README.md [hunk 2]
```diff
@@ -1,151 +1,185 @@
...
+              label: instance
+              matches: \d{1,3}(?:\.\d{1,3}){3}(?::\d{1,5})??
+              isIdentifier: true
+            - name: namespace
+              label: destination_service_namespace
+            - name: service
+              label: destination_service_name
+    ```
+
+  * Create a PrometheusServerConfig resource to sepcify configuration options for the Prometheus server. The following is an example of a [PrometheusServerConfig](https://github.com/turbonomic/turbo-metrics/blob/main/config/samples/metrics_v1alpha1_prometheusserverconfig_singlecluster.yaml) resource which specifies the location of the Prometheus server, and a label selector to exclude `jmx-tomcat` PrometheusQueryMapping resource:
```
Comment (amd-ibm) [inline deploy/README.md:153]: Misspelled specify as "sepcify"

**Review by amd-ibm — CHANGES_REQUESTED** [review]
Pease fix the spelling issue and I'll approve. Up to you if you want to address the nits.

File: README.md (hunk 1 above)
Comment (ading1977) [reply README.md]: fixed.

File: README.md [hunk 3]
```diff
@@ -1,47 +1,58 @@
...
-```
-
-
-# Deploy
-Follow the deployment instructions at [here](./deploy/) to deploy **Prometurbo** and **DIFProbe** container in the same Pod. 
+# Prometurbo
+
+Get metrics from [Prometheus](https://prometheus.io) for applications, and expose these applications and metrics in JSON
+format via REST API. The [`data ingestion framework probe`](https://github.com/turbonomic/data-ingestion-framework) (
+i.e., DIF Probe) will access the REST API, convert the JSON output to Turbonomic DTO to be consumed by Turbonomic
```
[the remark this answers is not in the document]
Comment (ading1977) [reply README.md]: fixed.

File: deploy/README.md (hunk 2 above)
Comment (ading1977) [reply deploy/README.md]: fixed.

**Review by amd-ibm — APPROVED** [review]

**Pull request merged**